auth/api/iam/openid4vp.go

```diff
@@ -832,18 +865,20 @@ func (r Wrapper) oauthNonceStore() storage.SessionStore {
 	return r.storageEngine.GetSessionDatabase().GetStore(oAuthFlowTimeout, oauthNonceKey...)
 }
 
-func assertParamPresent(params map[string]string, param ...string) error {
+func assertParamPresent(params map[string]interface{}, param ...string) error {
```

Comment:
oh here we have it, so just use this one instead of the new `for` range loops?

Reply:
fixed with  `singleStringParam`